src/jobs/yieldAccrualJob.ts

```
@@ -0,0 +1,33 @@
+    logger.info("Running initial yield accrual pass");
+    // Run one-day accrual to seed yields (good for seeded/test data)
+    await accrueFromStrategies(1, new Date());
+    logger.info("Initial yield accrual completed");
+  } catch (err) {
+    logger.error("Initial yield accrual failed", err);
+  }
+
+  // Daily schedule: pro-rate accrual by 1 day. Real production should run monthly or
+  // support configurable windows; this is intentionally conservative for now.
+  try {
+    setInterval(async () => {
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)